### PR TITLE
fix: default env to master

### PR DESCRIPTION
**Pull request by Khaledgarbaya**
This PR defaults the environment to master

Closes https://github.com/contentful/contentful-migration/issues/117

### CODE REVIEW

**Comment by connor-baer** [comment]
I wonder whether it would be better to force the user to provide an environment id. I can see myself accidentally running a migration on master with this change, simply because I forgot to provide the environment id.

**Comment by Khaledgarbaya** [comment]
Hey @connor-baer some users might not have access to the environment feature and all of our other tools are defaulting to master for now, This might change when we reach the v1.0.0

**Comment by zcei** [comment]
To adress @connor-baer's doubt, I'd really like to have a `Environment: <the-specified-environment>`  in the plan output. As the called URLs are relative to the environment you don't see what you're executing against during runtime.

**Comment by Khaledgarbaya** [comment]
@connor-baer @zcei we now have the current environment displayed in the plan
screen shot 2018-07-26 at 14 11 47

File: src/bin/lib/render-migration.ts [hunk 1]
```diff
@@ -19,8 +19,8 @@ const renderBatch = function (batch: RequestBatch) {
 }
 
 const renderPlan = (batches: RequestBatch[], envrionment: string) => {
-  console.log(chalk`{bold.underline Environment}: {bold.yellow ${envrionment}}\n`)
   console.log(chalk`{bold.green The following migration has been planned}\n`)
+  console.log(chalk`{bold.underline Environment}: {bold.yellow ${envrionment}}\n`)
```
Comment (connor-baer) [inline src/bin/lib/render-migration.ts]: Looks like you have a typo here: `${envrionment}` 😉

Comment (zcei) [reply src/bin/lib/render-migration.ts]: Haha, but thanks to auto-complete it's consistently wrong 😂

**Pull request merged**